rules/web/web_cve_2021_22005_vmware_file_upload.yml

```diff
@@ -0,0 +1,23 @@
+title: VMware vCenter Server File Upload CVE-2021-22005
+id: b014ea07-8ea0-4859-b517-50a4e5b7ecec
+status: experimental
+description: Detects exploitation attempts using file upload vulnerability CVE-2021-22005 in the VMWare vCenter Server.
+author: Sittikorn S
+date: 2021/09/24
+references:
+    - https://kb.vmware.com/s/article/85717
+    - https://www.tenable.com/blog/cve-2021-22005-critical-file-upload-vulnerability-in-vmware-vcenter-server
+tags:
+    - attack.initial_access
+    - attack.t1190
+logsource:
+    category: webserver
+detection:
+    selection:
+        cs-method: 'POST'
+        c-uri|contains:
+            - '/analytics/telemetry/ph/api/hyper/send?'
```

Comment:
As the list have only 1 value : `c-uri|contains: '/analytics/telemetry/ph/api/hyper/send?'`
Get less brackets when convert